src/components/gameScreen/components/customButton.js

```diff
@@ -1,15 +1,41 @@
-import React from "react";
-import { StyleSheet, Image, TouchableOpacity } from "react-native";
-
+import React, { useRef } from "react";
+import { StyleSheet, Image, TouchableOpacity, Animated } from "react-native";
+import tick from '../../../assets/img/icons/tick.png'
 export const CustomButton = ({
   color,
   img,
-  onTouch
+  onTouch,
+  buttonType
                            }) => {
+  const bookmarkScale = useRef(new Animated.Value(1)).current;
+  const tickScale = useRef(new Animated.Value(0)).current;
```

Comment:
You are using Animated from react native. Do you know what disadvantages it has comparing to react-native-reanimated?